tests/installation/system_role/select_role_KDE_desktop.pm

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Select System Role (Desktop with KDE Plasma)
+#          using REST API.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use scheduler 'get_test_suite_data';
+use testapi;
+
+sub run {
+    my $controller = $testapi::distri->get_system_role_controller();
```

Comment:
I suggest to use screen name (or logical unit name, e.g. as in case of expert_partitioner) as a variable for controller. This makes your test module to be more explicit in what it is doing and where. 

Let's imagine you need to call several controllers in your test module, having the variables called by meaningful names will make it more clear when reading the test module, then having 'controller1' do something, then 'controller2' do something. 

Something like, when one say "Select Desktop KDE system role on System Role Screen", in code it's more natural to call it like `$system_role->select_system_role('Desktop with KDE Plasma')`.

Other than that, LGTM. Thanks:)